plutus-validation-server/src/Main.hs

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main (main) where
+
+import           Data.Aeson
+import           Data.Aeson.Types
+import qualified Data.ByteString             as BS
+import           Data.ByteString.Base64.Type (getByteString64)
+import qualified Data.ByteString.Base64.Type as BS64
+import qualified Data.ByteString.Lazy        as BSL
+import           Network.HTTP.Types.Status
+import           Network.Wai
+import           Network.Wai.Handler.Warp
+
+main :: IO ()
+main = run 3000 app
+
+trueJSON :: BSL.ByteString
+trueJSON = "{\"isValid\":true}"
+
+falseJSON :: BSL.ByteString
+falseJSON = "{\"isValid\":false}"
+
+-- TODO: encoding: base16 or base64? Ask Vincent
+data ToValidate = ToValidate { _validator :: BS64.ByteString64
+                             , _redeemer  :: BS64.ByteString64
+                             , _data      :: BS64.ByteString64
+                             }
+
+instance FromJSON ToValidate where
+    parseJSON (Object v) = ToValidate
```

Comment:
Not the generic instance?

Reply:
Unfortunately, `data` is a keyword and so cannot be used.